src/domain/harbors/HarborsList.tsx

```diff
@@ -0,0 +1,141 @@
+        [styles.disabled]: disabled,
+      })}
+    >
+      {children}
+    </div>
+  );
+};
+
+type ColumnType = Column<HarborData> & { accessor: keyof HarborData };
+
+export interface HarborsPageProps {
+  data?: Array<HarborData>;
+}
+
+const HarborsList: React.FC<HarborsPageProps> = ({ data = [] }) => {
+  const { t } = useTranslation();
+
+  const columns: ColumnType[] = [
+    {
+      Cell: ({ cell }) => (
+        <InternalLink to={`/harbors/${cell.row.original.id}}`}>
+          {cell.value}
+        </InternalLink>
+      ),
+      Header: t('harbors.tableHeaders.harbor'),
+      accessor: 'name',
+    },
+    {
+      Header: t('harbors.tableHeaders.places'),
+      accessor: 'numberOfPlaces',
+    },
+    {
+      Cell: ({ cell }) => (
+        <IconOutline disabled={!cell.value}>
+          <IconPole
+            className={classNames(styles.icon, styles.outline, {
+              [styles.disabled]: !cell.value,
+            })}
+          />
+        </IconOutline>
+      ),
+      Header: () => <IconPole className={styles.icon} />,
+      accessor: 'electricity',
+    },
+    {
+      Cell: ({ cell }) => (
+        <IconOutline disabled={!cell.value}>
+          <IconFence
+            className={classNames(styles.icon, styles.outline, {
+              [styles.disabled]: !cell.value,
+            })}
+          />
+        </IconOutline>
+      ),
+      Header: () => <IconPole className={styles.icon} />,
+      accessor: 'gate',
+    },
+    {
+      Cell: ({ cell }) => (
+        <IconOutline disabled={!cell.value}>
+          <IconFence
+            className={classNames(styles.icon, styles.outline, {
+              [styles.disabled]: !cell.value,
+            })}
+          />
+        </IconOutline>
+      ),
+      Header: () => <IconPole className={styles.icon} />,
+      accessor: 'lighting',
+    },
+    {
+      Cell: ({ cell }) => (
+        <IconOutline disabled={!cell.value}>
+          <IconFence
+            className={classNames(styles.icon, styles.outline, {
+              [styles.disabled]: !cell.value,
+            })}
+          />
+        </IconOutline>
+      ),
+      Header: () => <IconWaterTap className={styles.icon} />,
+      accessor: 'water',
+    },
+    {
+      Cell: ({ cell }) => (
+        <IconOutline disabled={!cell.value}>
+          <IconFence
+            className={classNames(styles.icon, styles.outline, {
+              [styles.disabled]: !cell.value,
+            })}
+          />
+        </IconOutline>
+      ),
+      Header: () => <IconTrash className={styles.icon} />,
+      accessor: 'wasteCollection',
+    },
+  ];
+
+  return (
+    <div className={styles.harborsPage}>
+      <Table
+        data={data}
+        columns={columns}
+        renderSubComponent={row => <HarborDetails {...row.original} />}
+        renderMainHeader={() => t('harbors.tableHeaders.mainHeader')}
+        canSelectRows
+      />
+    </div>
+  );
+};
+
+export default HarborsList;
```

Comment:
Why do we need this new component in this PR?

Reply:
HarborsPageContainer and HarborsPage have been reorganized into HarborsList and HarborsPageContainer. This enables us to do UI development in storybook without extra dependencies.